src/layers/arp.rs

```diff
@@ -66,6 +66,10 @@ impl Layer for ARP {
         Ok((None, ARP_HEADER_LENGTH))
     }
 
+    fn stack_and_encode(&mut self, _next_layer: &[u8], _info: &str) -> Result<Vec<u8>, Error> {
```

Comment:
This and other `Layer`s where it is `todo!()`, let's just get rid of those from the PR, this will make the PR considerably shorter and easier to review!!

Sorry I missed this earlier.